service-registry.yaml

```diff
@@ -183,13 +183,9 @@ components:
         description:
           type: string
           description: A human readable general description of this service.
-        metadata:
+        extension:
           type: object
-          additionalProperties:
-            type: string
-          description: >-
-            A set of key-value pairs that represent metadata specific to this
-            service.
+          description: A sub-object used by implementing services to provide additional data about the service. Extensions are not compatible between implementations, but should be consistent between services sharing a single identifier.
```

Comment:
I don't understand this description. E.g. Sub-object? Extensions are not compatible between implementations? Sharing a single identifier? Is there a better way to phrase this?

Reply:
Yeah little bit too chatty. Please have a look now